doc/source/developer_guide/dg_forcing.rst

```diff
@@ -0,0 +1,216 @@
+- Qa      = specific humidity (kg/kg)
+- flw     = incoming longwave radiation (W/m^2)
+- fsw     = incoming shortwave radiation (W/m^2)
+- fsnow   = snowfall rate (kg/m^2 s)
+
+and model forcing inputs are derived from those fields and the defaults.
+
+Because the input files are on the gregorian time axis, the model can run with the regular
+365 day (noleap) calendar, but in that case, the Feb 29 input data will be used on 
+March 1, and all data
+after March 1 will be shifted one day.  December 31 in leap years will be skipped when
+running with a CICE calendar with no leap days.
+
+
+.. _NCARforcing:
+
+NCAR Atmosphere Forcing
+-------------------------
+
+The NCAR atmospheric forcing was used in earlier standalone runs on the gx3 grid, and the
+Consortium continues to do some limited testing with this forcing dataset.
+Monthly average data for fsw, cldf, fsnow are read.  6-hourly data for
+Tair, uatm, vatm, rhoa, and Qa are also read.
+Users are encouraged to switch to the JRA55 (see :ref:`JRA55forcing`) dataset.  This
+atmosphere forcing dataset may be deprecated in the future.
+
+
+.. _LYqforcing:
+
+LYq Atmosphere Forcing
+-------------------------
+
+The LYq (Large and Yeager :cite:`Large09`) forcing was used in earlier standalone 
+runs on the gx1 grid, and the
+Consortium continues to do some very limited testing with this forcing dataset.
+This dataset is largely based on the CORE II data.
+Monthly average data for cldf and fsnow is read while 6-hourly data for Qa, Tair, 
+uatm, and vatm are read with other fields derived or set by default.
+Users are encouraged to switch to the JRA55 (see :ref:`JRA55forcing`) dataset. This
+atmosphere forcing dataset may be deprecated in the future.
+
+
+.. _defaultforcing:
+
+Default Atmosphere Forcing
+----------------------------
+
+The default atmosphere forcing option sets the atmosphere forcing
+internally.  No files are read.  Values for forcing fields are defined
+at initialization in subroutine **init_coupler_flux** and held
+constant thereafter.  Different conditions can be specified thru the
+``default_season`` namelist variable.
+
+
+.. _box2001forcing:
+
+Box2001 Atmosphere Forcing
+-------------------------
+
+The box2001 forcing dataset in generated internally.  No files are read.  The
+dataset is used to test an idealized box case as defined in :cite:`Hunke01`.
+
+
+.. _otheratmforcing:
+
+Other Atmosphere Forcing
+-------------------------
+
+There are a few other atmospheric forcing modes, as defined by ``atm_data_type``, but
+they are not tested by the Consortium on a regular basis.
+
+
+.. _defaultocnforcing:
+
+Default Ocean Forcing
+-------------------------
+
+The ``default`` ocean setting is the standard setting used in standalone CICE runs.
+In this mode, the sea surface salinity is set to 34 ppt and the sea surface
+temperature is set to the freezing temperature at all grid points and
+held constant.  Other ocean coupling fields are set to zero.  No files are read.
```

Comment:
held constant unless the mixed layer parameterization is turned on, in which case the SST evolves

Reply:
and when the mixed layer parameterization is on, it can use data from the ocean forcing file, which we only have for gx1.  It might not be used in our testing and might not be offered as a forcing data set...

Reply:
I updated the text a bit, but did not add the stuff about an ocean forcing file.  I don't think we test anything other than the default, and I don't think we've released any ocean forcing dataset.